Modules/unicodedata.c

```diff
@@ -681,15 +681,15 @@ nfc_nfkc(PyObject *self, PyObject *input, int k)
       if (LBase <= code && code < (LBase+LCount) &&
           i + 1 < len &&
           VBase <= PyUnicode_READ(kind, data, i+1) &&
-          PyUnicode_READ(kind, data, i+1) <= (VBase+VCount)) {
+          PyUnicode_READ(kind, data, i+1) < (VBase+VCount)) {
           int LIndex, VIndex;
           LIndex = code - LBase;
           VIndex = PyUnicode_READ(kind, data, i+1) - VBase;
           code = SBase + (LIndex*VCount+VIndex)*TCount;
           i+=2;
           if (i < len &&
-              TBase <= PyUnicode_READ(kind, data, i) &&
-              PyUnicode_READ(kind, data, i) <= (TBase+TCount)) {
+              TBase < PyUnicode_READ(kind, data, i) &&
```

Comment:
Let me give a supplement:

Before Unicode 4.1.0 (draft), here is: `TBase <= code <= TBase + TCount`  
see: [http://www.unicode.org/reports/tr15/tr15-**24**.html#hangul_composition](http://www.unicode.org/reports/tr15/tr15-24.html#hangul_composition)

After Unicode 4.1.0, here is `TBase < code < TBase + TCount`, which in line with the latest version (Unicode 10.0)  
see: [http://www.unicode.org/reports/tr15/tr15-**25**.html#hangul_composition](http://www.unicode.org/reports/tr15/tr15-25.html#hangul_composition)

This change happened in 2005.